### ISSUE TITLE
Troubles opening the data set

### ISSUE THREAD

**Issue by SytseGroenwold** [marcoortu/jira-social-repository#1]
I tried opening the data set in a few ways. 
First, I used sqlite3 and tried to import the data using DB Browser for SQLite. I did not get any messages, but no data. When running the script I did however get a pop up to enter a passphrase due to encryption.
Secondly, I downloaded MySQL and MySQL Workbench. In the latter I used File > Open SQL Script. When importing the data, I get some generic error. When selecing "Run SQL script", I get a more specific error: "Error executing SQL script.
'>' not supported between instances of 'NoneType' and 'int'".

Not sure if this is something others have come across and been able to resolve?
Maybe relevant: I am on MacOS and the extract.sh script did not work for me, I had to copy and paste the one-liner in that file straight into my terminal for it to work. Maybe something went wrong in this approach?

**Comment by Mobtade2** [comment]
I am unable to open the 4.51GB SQL file. I tried to open it with several drivers (MySQL, SQL, SQLite), uploading it to Google cloud SQL and connecting to it, yet I was unable to reach the data. Any help would be highly appreciated. What is the driver?

**Comment by Mobtade2** [comment]
@SytseGroenwold  I managed to access the data. I am also using a mac btw.
Regarding your .sh problem, I remember that I did a quick research online and managed to solve it, it was something about access rights. The resulting file is a Postgres SQL DB v. 9.4. 
I tried to upload it to Google Cloud SQL but it did not work, so I had to set up a local Postgres instance on my mac (you will find many sources online on how to do that). Then I used the following command in the terminal to import the SQL file into the created DB

psql (DB instance name) <  (SQL file location) --user (user account to access the DB)

you will be prompted for the password that you used to create the account, then it will take some time as it's importing each row (in my old 2015 machine it took about 2 hours). 
Good luck :)
